### PR TITLE
Lmdevs 364 add calculator

### PR DESCRIPTION
**Pull request by Steven-Yulong-Yan**
### Motivation

Create a calculator for the advanced trading interface.

### Changes

- Added a calculator to the trading panel
- Fulfilled basic calculating functionalities

### CODE REVIEW

File: components/Trade/Advanced/TradingPanel/TradingInput.tsx [hunk 1]
```diff
@@ -133,14 +132,14 @@ export default styled(({ selectedTracer, className, account }: TIProps) => {
...
                 {/* Dont display these if it is a limit order*/}
-                {order?.orderType !== 1 ? (
-                    <>
-                        {/* Leverage select */}
-                        <Leverage leverage={order?.leverage ?? 1} />
-                    </>
-                ) : (
-                    <></>
-                )}
+                {/*{order?.orderType !== 1 ? (*/}
```
Comment (dospore) [inline components/Trade/Advanced/TradingPanel/TradingInput.tsx:135]: Was this intentionally removed??

File: components/Trade/Advanced/TradingPanel/TradingInput.tsx [hunk 2]
```diff
@@ -76,30 +75,30 @@ const OrderTypeSelect: React.FC<SProps> = styled(({ selected, className }: SProp
...
-    display: flex;
-
-    > .label {
-        margin: 5px auto 35px 0;
-        font-size: 16px;
-        letter-spacing: -0.32px;
-        color: #3da8f5;
-    }
-`;
+// type LProps = {
```
Comment (dospore) [inline components/Trade/Advanced/TradingPanel/TradingInput.tsx:78]: Was this intentionally removed??

File: components/Nav/Navbar/index.tsx [hunk 3]
```diff
@@ -171,9 +171,8 @@ const DropdownLogo = styled(({ className }) => {
     }
 
     > .dropdown-menu {
-        opacity: 0;
         position: absolute;
-        display: block;
+        display: none;
```
Comment (dospore) [inline components/Nav/Navbar/index.tsx:175]: Cant we increase/decrease z-index on the nav to fix this without using display none??

File: components/Trade/Advanced/TradingPanel/TradingInput.tsx (hunk 1 above)
Comment (Steven-Yulong-Yan) [reply components/Trade/Advanced/TradingPanel/TradingInput.tsx:135]: The new leverage slider will take its place here eventually

File: components/Nav/Navbar/index.tsx (hunk 3 above)
Comment (Steven-Yulong-Yan) [reply components/Nav/Navbar/index.tsx:175]: Will fix

**Review by dospore — APPROVED** [review]

**Pull request merged**